### PR TITLE
run JuliaFormatter on Base64 stdlib

### PR DESCRIPTION
**Pull request by Moelf**
part of:
- https://github.com/JuliaLang/julia/pull/47094

this is one of the smallest stdlib with some interesting patterns. We can use this to test/suggest
features (mostly, how to opt-out of certain formatting) to JuliaFormatter

cc. @domluna

### CODE REVIEW

File: stdlib/Base64/src/encode.jl [hunk 1]
```diff
@@ -69,10 +69,10 @@ function Base.unsafe_write(pipe::Base64EncodePipe, ptr::Ptr{UInt8}, n::UInt)::In
     i = 0
     p_end = ptr + n
     while true
-        buffer[i+1] = encode(b1 >> 2          )
```
Comment (Moelf) [inline stdlib/Base64/src/encode.jl:72]: these are examples of unwanted alignment change

this can also happen at location of `=` instead of ()

**Comment by ViralBShah** [comment]
It seems that the Formatter here is making some things worse, which were hand formatted for readability.

**Comment by Moelf** [comment]
indeed. we're waiting for upstream to make more things "opt in": https://github.com/domluna/JuliaFormatter.jl/issues/561

**Comment by ViralBShah** [comment]
Close this for now and reopen when upstream addresses the formatting issues?

**Comment by Moelf** [comment]
why not leave it open?

**Pull request closed without merging**